scripts/install_local_k8s.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+CHANGE_MINIKUBE_NONE_USER=true
+
+curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.8.0/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
+sudo minikube start --vm-driver=none --kubernetes-version=v1.8.0
+minikube update-context
+JSONPATH='{range .items[*]}{@.metadata.name}:{range @.status.conditions[*]}{@.type}={@.status};{end}{end}'
+until kubectl get nodes -o jsonpath="$JSONPATH" 2>&1 | grep -q "Ready=True"; do sleep 1; done
```

Comment:
Can you add some comments here? I barely know what is going on.

Reply:
Comments added.